server/routes/paymentPlan.routes.js

```diff
@@ -0,0 +1,85 @@
+import express from 'express';
+import { authenticate, schemaValidation, isAdmin, hasValidObjectId } from '../helpers/middleware';
+import addPlanSchema from '../schemas/paymentPlan.schema';
```

Comment:
`addPlanSchema` is not explicit enough. The codebase is getting bigger, it is easier to mix things up now.
```suggestion
import addPaymentPlanSchema from '../schemas/paymentPlan.schema';
```